app.rb

```diff
@@ -0,0 +1,85 @@
+### Arrays
+
+# Starting with the following array...
+
+
+planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
+
+
+# Access the second value in `planeteers`.
+
+
+planeteers[1]
+
+
+# Add "Heart" to the end of `planeteers`.
+
+
+planeteers.push("Heart")
+
+
+# Remove "Captain Planet" from the array (without using a method).
```

Comment:
to remove  "Captain Planet" from the array without using a method you can subtract it from the array
` planeteers  - ["Captain Planet"]`